terraform-apply.sh

```diff
@@ -55,8 +40,12 @@ ${TERRAFORM} get \
 if [ "${TERRAFORM_ACTION}" = "plan" ]; then
   ${TERRAFORM} $TERRAFORM_ACTION \
     -refresh=true \
-    -out="${PLAN_FILE:-}" \
+    -out=./terraform-state/terraform.tfplan \
     $DIR
+
+  ${TERRAFORM} show ./terraform-state/terraform.tfplan \
+    | grep -v "This plan does nothing." \
```

Comment:
I think this will fail when running with `-e` when the plan is empty, as the grep won't find anything and exit 1

```
$ echo "The plan does nothing." | grep -v "The plan does nothing."; echo $?
1
```